Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.py

```diff
@@ -0,0 +1,613 @@
+
+    data = remove_empty_elements({
+        "startDateTime": start_time,
+        "endDateTime": end_time,
+        "recordLimit": limit,
+        "subject": {
+            "ipAddresses": {
+                "includes": ip_addresses if isinstance(ip_addresses, list) else [ip_addresses]
+            }
+        }
+    })
+    response = client.initialize_flow_search(tenant_id, data)
+    outputs = dict_safe_get(response, ['data', 'query'])
+    table = tableToMarkdown('Query Flows Initializing Information:', outputs,
+                            headers=['id', 'status', 'percentComplete'], removeNull=True)
+    return CommandResults(
+        outputs_prefix='CiscoStealthwatch.FlowStatus',
+        outputs_key_field='id',
+        raw_response=response,
+        outputs=outputs,
+        readable_output=table)
+
+
+def cisco_stealthwatch_query_flows_status_command(client: Client, tenant_id: str,
+                                                  search_id: str) -> CommandResults:
+    """Retrieve query flow status using search id
+
+    Args:
+        client (Client): Cisco Stealthwatch Client
+        tenant_id (str): The id of the tenant the search was performed on
+        search_id (str): The id of the search
+
+    Returns:
+        CommandResults: Raw response, outputs and readable outputs
+    """
+    response = client.check_flow_search_progress(tenant_id, search_id)
+    outputs = dict_safe_get(response, ['data', 'query'])
+    outputs['id'] = search_id
+    table = tableToMarkdown('Query Flows Status Information:', outputs,
+                            headers=['id', 'percentComplete'], removeNull=True)
+    return CommandResults(
+        outputs_prefix='CiscoStealthwatch.FlowStatus',
+        outputs_key_field='id',
+        raw_response=response,
+        outputs=outputs,
+        readable_output=table)
+
+
+def cisco_stealthwatch_query_flows_results_command(client: Client, tenant_id: str,
+                                                   search_id: str) -> CommandResults:
+    """Retrieve the results for a query flow by search id
+
+    Args:
+        client (Client): Cisco Stealthwatch Client
+        tenant_id (str): The id of the tenant the search was performed on
+        search_id (str): The id of the search
+
+    Returns:
+        CommandResults: Raw response, outputs and readable outputs
+    """
+    response = client.get_flow_search_results(tenant_id, search_id)
+    outputs = []
+    for data in dict_safe_get(response, ['data', 'flows']):
+        outputs.append(data)
+    headers = ['id', 'tenantId', 'flowCollectorId', 'protocol', 'serviceId', 'statistics', 'peer',
+               'subject']
+    table = tableToMarkdown('Query Flows Results Information:', outputs, headers=headers,
+                            removeNull=True)
+    return CommandResults(
+        outputs_prefix='CiscoStealthwatch.FlowResults',
+        outputs_key_field=['id', 'tenantId'],
```

Comment:
The id alone is not enough? did you check for duplicates?

Reply:
I re-checked it and the `id` here is enough, but on other commands, like `cisco_stealthwatch_get_top_alarming_tags_command` and `cisco_stealthwatch_get_tag_hourly_traffic_report_command`, there is no id to rely on, so I used a combination of the `tag id`, `tenant id`, and the `timestamp` to make sure there is no missing information